core/common/src/main/java/alluxio/grpc/GrpcChannel.java

```diff
@@ -106,7 +97,11 @@ public synchronized void shutdown() {
       mAuthDriver = null;
     }
     if (!mChannelReleased) {
-      GrpcManagedChannelPool.INSTANCE().releaseManagedChannel(mChannelKey, mShutdownTimeoutMs);
+      try {
+        mConnection.close();
+      } catch (Exception e) {
```

Comment:
why is this conversion necessary? maybe add a comment

Reply:
which conversion?

Reply:
It boils down to what do we want to do with failed release attempts. 
I believe It's good to have an exception with a stack-trace at least until stabilized.